bokehjs/package.json

```diff
@@ -79,6 +79,7 @@
     "@bokeh/numbro": "^1.6.2",
     "@bokeh/slickgrid": "~2.4.2701",
     "choices.js": "^9.0.1",
+    "ci": "^1.0.0",
```

Comment:
This seems an unnecessary dependency. If you're using this in your own workflow, you should `npm install -g ci`.

Reply:
Yes, it's a mistake. I need to go through the package changes carefully as there are some I added that I later decided I didn't need but I may not have removed them fully.